Makefile

```diff
@@ -69,6 +71,24 @@ install: kubectl-aks
 	mkdir -p ~/.local/bin/
 	cp kubectl-aks ~/.local/bin/
 
+# Uninstall
+.PHONY: uninstall
+uninstall:
+	rm -f ~/.local/bin/kubectl-aks
+
+# Install innovation engine
```

Comment:
I suggest moving this to another Makefile as it is not compiling or testing anything related to kubectl-aks.